src/gitpodify.ts

```diff
@@ -15,91 +16,91 @@ let observer: MutationObserver | undefined;
  * or on the active tab when the user clicked on the Gitpod icon.
  */
 const init = async (injectedByUserClick: boolean = false) => {
-    const configProvider = await ConfigProvider.create();
-    const config = configProvider.getConfig();
+  const configProvider = await ConfigProvider.create();
+  const config = configProvider.getConfig();
 
-    // Tell every page that looks like a Gitpod installation that the browser extension is installed!
-    const gitpodExtensionCheckElement = document.getElementById('ExtensionCheck_GitpodBrowserExtension');
-    if (gitpodExtensionCheckElement) {
-        gitpodExtensionCheckElement.innerHTML = "installed";
-        if (injectedByUserClick) {
-            window.open(config.gitpodURL);
-        }
-        // This page is a Gitpod page. We are done.
-        return;
-    }
-
-    // Get the first injector that canHandleCurrentPage()
-    const injectorProvider = new InjectorProvider(configProvider);
-    const injector = injectorProvider.findInjectorForCurrentPage();
-    if (!injector) {
-        // We do not have an injector for this page. We just open Gitpod.
-        if (injectedByUserClick) {
-            window.open(config.gitpodURL);
-        }
-        return
+  // Tell every page that looks like a Gitpod installation that the browser extension is installed!
+  const gitpodExtensionCheckElement = document.getElementById(
+    "ExtensionCheck_GitpodBrowserExtension"
+  );
+  if (gitpodExtensionCheckElement) {
+    gitpodExtensionCheckElement.innerHTML = "installed";
+    if (injectedByUserClick) {
+      browser.tabs.create(config.gitpodURL);
     }
+    // This page is a Gitpod page. We are done.
+    return;
+  }
 
+  // Get the first injector that canHandleCurrentPage()
+  const injectorProvider = new InjectorProvider(configProvider);
+  const injector = injectorProvider.findInjectorForCurrentPage();
+  if (!injector) {
+    // We do not have an injector for this page. We just open Gitpod.
     if (injectedByUserClick) {
-        // User clicked on the Gitpod extension icon. We open the Gitpod with this page as context.
-        window.open(renderGitpodUrl(config.gitpodURL));
+      browser.tabs.create(config.gitpodURL);
     }
+    return;
+  }
+
+  //   BREAKING, TODO: Fix this
+  if (injectedByUserClick) {
+    // User clicked on the Gitpod extension icon. We open the Gitpod with this page as context.
+    browser.tabs.create(renderGitpodUrl(config.gitpodURL));
+  }
```

Comment:
related to https://github.com/gitpod-io/browser-extension/pull/65#discussion_r1064388992